contribute/developer-guide/README.md

```diff
@@ -238,7 +238,8 @@ Follow the steps provided below to setup okteto & test the experiment execution.
       'experiments/sample_category/sample_exec_chaos',
       'experiments/sample_category/sample_exec_chaos/experiment',
       ```
-    - Add `&` operator in the end of chaos-inject command `CHAOS_INJECT_COMMAND` example: `md5sum /dev/zero &`. As we are running chaos command as a background process in a separate thread.  
+    - Add `&` operator at the end of chaos commands `CHAOS_INJECT_COMMAND` and `CHAOS_KILL_COMMAND` example: `md5sum /dev/zero &`. 
```

Comment:
Do we need `&` for CHAOS_KILL_COMMAND?

Reply:
Yes, we need it for multiple target pods, Like to get execute and to get back to another target thread.